packages/sdk-react/src/components/onboarding/types.ts

```diff
@@ -116,8 +101,7 @@ export type OrganizationRequirements = Partial<
 
 export type OnboardingPersonIndex = string | null;
 
-export type OnboardingRequirementMask =
-  | OnboardingRequirement.REPRESENTATIVE
-  | OnboardingRequirement.DIRECTORS
-  | OnboardingRequirement.OWNERS
-  | OnboardingRequirement.EXECUTIVES;
+export type OnboardingRequirementMask = Extract<
+  components['schemas']['OnboardingRequirement'],
+  'representative' | 'directors' | 'executives'
```

Comment:
I think the owners are missed. It should be
`'representative' | 'directors' | 'executives' | 'owners'`

Reply:
🔬 Fixed!